### ISSUE TITLE
"browser-specific" Navigator properties are complicated

### ISSUE THREAD

**Issue by bzbarsky** [GoogleChrome/confluence#238]
There are properties that the HTML spec requires to be on Navigator only for some values of the UA string (required for web compat).  That means that while those properties may only be present in UAs that have that UA string, they are not, in some sense, "browser-specific"; they are UA-string-specific....

**Comment by foolip** [comment]
Yes, that's unfortunately case, and we even enshrined it in the spec:
https://html.spec.whatwg.org/multipage/system-state.html#concept-navigator-compatibility-mode

@bzbarsky, do you think that https://github.com/GoogleChromeLabs/confluence/issues/219 is the remedy, or something else?

**Comment by bzbarsky** [comment]
I don't know; hence "complicated".
